descriptors/ModuleDescriptor-template.json

```diff
@@ -136,7 +139,10 @@
             "orders-storage.po-lines.item.post",
             "orders-storage.alerts.item.post",
             "orders-storage.reporting-codes.item.post",
-            "configuration.entries.collection.get"
+            "configuration.entries.collection.get",
+            "acquisitions-units-storage.units.collection.get",
+            "acquisitions-units-storage.memberships.collection.get",
+            "orders-storage.acquisitions-unit-assignments.collection.get"
```

Comment:
we are no longer going to use "orders-storage.acquisitions-unit-assignments.collection.get"

Reply:
deleted